dietstory-routing/registration/validate_forms.py

```diff
@@ -0,0 +1,42 @@
+from django import forms
+from datetime import date
+from django.core.validators import validate_email, validate_slug, BaseValidator
+
+
+def calculate_age(birthday):
+    today = date.today()
+    return today.year - birthday.year - ((today.month, today.day) < (birthday.month, birthday.day))
+
```

Comment:
make `calculate_age` a member function (or static function of) of `MinAgeValidator` if only used there

Reply:
Was planning to make that function used by more than one class, but for now MinAgeValidator is the only one using it. calculate_age is moved into the MinAgeValidator class now.